R/cache.R

```diff
@@ -271,7 +271,7 @@ renv_cache_diagnose_corrupt_metadata <- function(paths, problems, verbose) {
 
     # nocov start
     if (verbose) {
-      renv_pretty_print(
+      notice(
```

Comment:
Each of these feel like they should not be subject to the verbose control because it is reporting specific instances of corrupt metadata. Is there a reason for the caller to suppress this output? Is that rare enough that they could use `capture.output`?